javascript/ql/test/query-tests/Security/CWE-089/untyped/pg-promise-types.ts

```diff
@@ -0,0 +1,13 @@
+import { IDatabase } from "pg-promise";
```

Comment:
nit: this file belongs to the `typed` directory

Reply:
The `typed` directory is for tests that rely on type information provided by the TypeScript compiler, which in turn requires a `.d.ts` shim.

As of https://github.com/github/codeql/pull/4401, however, we don't need that in our tests anymore, which is why the test works in this directory.